### PR TITLE
Load existing credit cards and allow payment with them

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Load existing credit cards and allow payment with them

[stage-2]

## Motivation and Context
Billing Frictions epic

## How Has This Been Tested?
Tested changes locally. Will update unit tests in a separate PR.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
Initial PR. Will update unit tests separately.

@ezequielc @santiagonoguez please review. Thanks!

File: web/partials/purchase/credit-card-form.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
   <!-- NEW CREDIT CARD -->
   <div id="new-credit-card-form" ng-show="factory.paymentMethods.selectedCard.isNew">
-    <div class="row">
+    <div class="row" ng-if="factory.paymentMethods.selectedCard.isNew">
```
Comment (alex-deaconu) [inline web/partials/purchase/credit-card-form.html:16]: Removing from dom to prevent validation errors when submitting the form.

Comment (ezequielc) [reply web/partials/purchase/credit-card-form.html:16]: @alex-deaconu To confirm, are you removing only Name field? Wouldn't it work if you change `ng-show` above to `ng-if` and remove all fields? Thanks

**Review by ezequielc — APPROVED** [review]
@alex-deaconu Looks good 👍 
I noticed that there is no way to navigate back to credit cards list once you choose to enter a new card, there is nothing in the mockups. Something to improve down the road. Thanks

File: web/partials/purchase/credit-card-form.html (hunk 1 above)
Comment (alex-deaconu) [reply web/partials/purchase/credit-card-form.html:16]: @ezequielc good question. In order to preserve the values in the Stripe CC fields for purchase-flow we cannot use `ng-if` for the whole form. Thanks.

**Comment by alex-deaconu** [comment]
@ezequielc yeah unfortunately there's no UI for that. I thought about making the "Cancel" button work in that manner but it's not very intuitive. Definitely something we can easily improve. Thanks.

File: web/partials/purchase/credit-card-form.html (hunk 1 above)
Comment (ezequielc) [reply web/partials/purchase/credit-card-form.html:16]: Got it. Thanks!

**Pull request merged**